### PR TITLE
i already did it

### PR DESCRIPTION
**Pull request by RomSmile**
good luck to check!

### CODE REVIEW

**Comment by danheim** [comment]
add demo link and test report link

**Review by danheim — CHANGES_REQUESTED** [review]
Do npm run deploy and add demo link to the description

**Comment by RomSmile** [comment]
DEMO LINK: https://RomSmile.github.io/layout_search-bar-airbnb/
TEST REPORT LINK: https://RomSmile.github.io/layout_search-bar-airbnb/report/html_report/

**Review by varseniuk — CHANGES_REQUESTED** [review]
Your page and test report still are not deployed to github pages.
Have you done `npm run update` and `npm run deploy`?

**Comment by RomSmile** [comment]
DEMO LINK: https://RomSmile.github.io/layout_search-bar-airbnb/
TEST REPORT LINK: https://RomSmile.github.io/layout_search-bar-airbnb/report/html_report/

**Comment by RomSmile** [comment]
Вроде написал две команды npm run update затем npm run deploy. Но ссылки всё равно не работают..

File: src/index.html [hunk 1]
```diff
@@ -11,6 +12,30 @@
...
+    <form
+      data-qa="big"
+      method="POST"
+      class="form_for_search"
+    >
+      <input
+        type="text"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="first_search_bar bar"
```
Comment (varseniuk) [inline src/index.html:24]:
```suggestion
        class="large_search_bar bar"
```
It's not clear what `first` or `second` should mean

**Pull request closed without merging**